agent/config/default.go

```diff
@@ -122,6 +122,8 @@ func DefaultSource() Source {
 			proxy_max_port = 20255
 			sidecar_min_port = 21000
 			sidecar_max_port = 21255
+			expose_min_port = 21500
```

Comment:
If any of your tests rely upon having actual listeners bind to these ports to avoid test flakes you should probably ensure the test agents use `freeport` to allocate the range. There are two places this matters:
```
consul:master $ git grep 'freeport.MustTake.*6'
agent/testagent.go:     ports := freeport.MustTake(6)
sdk/testutil/server.go: ports := freeport.MustTake(6)
```

I didn't find anything yet that would indicate this is necessary, just pointing it out.

Reply:
currently don't have tests for that, will go in with Envoy integration tests